internal/signerverifier/ssh/ssh.go

```diff
@@ -174,3 +176,30 @@ func parseSSH2Key(data string) (ssh.PublicKey, error) {
 	body := strings.Join(lines[i:], "")
 	return parseSSH2Body(body)
 }
+
+// KeyToSSlibKey converts an ssh Key into an SSlibKey.
+func KeyToSSlibKey(key *Key) *sv.SSLibKey {
```

Comment:
is there a reason for this to not be a method on `Key`? `key.SSLibKey()`

Reply:
No reason. I can change it.